base/math.jl

```diff
@@ -826,6 +817,7 @@ julia> rem2pi(7pi/4, RoundDown)
 5.497787143782138
 ```
 """
+
```

Comment:
preferable not to have any empty lines between a docstring and the signature it's documenting